static/hosting-intro/bashconsole.js

```diff
@@ -0,0 +1,22 @@
+$(function() {
+  function swapText() {
+    var strings = [
+    "pip install syncano-cli",
+    "syncano login --instance-name YOUR_INSTANCE",
+    "syncano hosting list",
+    "syncano hosting list files",
+    "syncano hosting publish BASE_DIR",
+    "syncano hosting delete",
+    "syncano hosting delete hosting/file/path",
+    "syncano hosting update hosting/file/path local/file/path",
+    "syncano hosting --domain staging publish BASE_DIR",
+    "syncano hosting publish ./your/project/path/"]
+
+    $(".element").typed({
```

Comment:
1. store element in variable to prevent searching in DOM with every function call, 2. use better class name for selector, 3. use single quote